tests/storage/test_database.py

```diff
@@ -37,6 +38,101 @@ def test_native_tuple_comparison(self) -> None:
         self.assertEqual(args, [1, 2])
 
 
+class ExecuteScriptTestCase(unittest.HomeserverTestCase):
+    """Tests for `BaseDatabaseEngine.executescript` implementations."""
```

Comment:
To check: will this get run against both postgres and sqlite in CI?

Reply:
Yep, that's the intention.

Reply:
CI run for sqlite: https://github.com/matrix-org/synapse/actions/runs/4018312895/jobs/6903837184#step:9:3081
CI run for postgres: https://github.com/matrix-org/synapse/actions/runs/4018312895/jobs/6903837341#step:9:3174